src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/StartlistTimer.razor

```diff
@@ -8,11 +9,11 @@
 
 @code {
     [Parameter]
-    public string StartTime { get; set; } = default!;
+    public TimeSpan StartTime { get; set; } = default!;
     [Parameter]
     public double StopTimerAtTime { get; set; } = default!;
 
-    const int TimerRefreshInMilliseconds = 1000;
+    TimeSpan TimerRefreshTime = TimeSpan.FromSeconds(1);
```

Comment:
`TimeSpan` represents an interval of time, rather than a point in time, so naming it time isn't very helpful. I'd do `TimerInterval`.